user_guide.md

```diff
@@ -77,6 +83,36 @@ At any time you can inspect the kubernetes objects definitions for a particular
 ks show ${KF_ENV} -c kubeflow-core
 ```
 
+### Usage Reporting
+
+By default, Kubeflow will report **anonymous** usage data using [spartakus](https://github.com/kubernetes-incubator/spartakus), Kubernetes' reporting tool. Spartakus **does not report any personal information**. See [here](https://github.com/kubernetes-incubator/spartakus) for more detail.
+This is entirely voluntary and you can opt out by doing the following
+
+```
+ks param set kubeflow-core reportUsage false
+
+# Delete any existing deployments of spartakus
+kubectl delete -n ${NAMESPACE} deploy spartakus-volunteer
+```
+
+To explictly enable usage reporting repeat the above steps setting reportUsage to `true`
+
+```
+ks param set kubeflow-core reportUsage true
+
+# Delete any existing deployments of spartakus
+kubectl delete -n ${NAMESPACE} deploy spartakus-volunteer
+```
+
+**Reporting usage data is one of the most signifcant contributions you can make to Kubeflow; so please consider leaving it on.** This data
```

Comment:
maybe instead of "consider leaving it on" say "consider turning it on", as this changed to opt-in